rfc/rfc022-discovery-service.md

```diff
@@ -74,40 +79,43 @@ The Verifiable Presentation MUST NOT be valid longer than the Verifiable Credent
 
 Clients MUST read the list by sending an HTTP GET to the server's service endpoint.
 The server MUST return a 200 OK response with a JSON object containing:
-- `entries` REQUIRED. MUST contain a JSON array with zero or more presentations.
-- `tag` REQUIRED. MUST be a JSON string referring to the last entry. The tag is an opaque value; the client SHOULD NOT derive any meaning from it.
+- `entries` REQUIRED. MUST contain a JSON object with a mapping of timestamp (as string) to presentation.
```

Comment:
why is it a string? Also, this causes out-of-order map entries due to map serialization semantics, thus requires sorting.

Also, since the LC is monotonically increasing (thus, no gaps), you can simply return the first and last LCs.
Then also no sorting on the client side is required to make sure it processes the VPs in the ascending LC value order.

We might not need starting LC per se, but it could be by the client to determine out-of-syncness. And just for completeness

Reply:
There could be holes: client submits two VPs, latest wins.
Using the key from the mapping as LC value for the client makes it deterministic and requires no sorting.

Reply:
An entry need not to be a mapping, it could also be a object with timestamp and vp fields.